sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -313,10 +353,15 @@ Table {counter: tableNumber}. Device Support Aspect.
 |`aspect::ext_oneapi_graph`
 | Indicates that the device supports all the APIs described in this extension.
 |`aspect::ext_oneapi_limited_graph`
-| Indicates that the device supports all the APIs described in this extension
-except for those described in the <<executable-graph-update, Executable Graph
-Update>> section. This is a temporary aspect that we intend to remove once
-devices with full graph support are more prevalent.
+a| Indicates that the device supports all the APIs described in this extension
```

Comment:
```suggestion
| Indicates that the device supports all the APIs described in this extension
```

Doesn't seem to affect the rendering, but sneaky `a`

Reply:
Ah this is actually to indicate that there are things in the table cell that require ASCIIDOC rendering, either the links or the list I can't remember which.